### ISSUE TITLE
Notifications requests NotificationGroupManager / Class initialization must not depend on services

### ISSUE THREAD

**Issue by ahus1** [jshiell/checkstyle-idea#655]
I receive this exception on version 5.96.0 on the following IntelliJ version - please have a look when you have the time. Thanks!

IntelliJ IDEA 2024.2.4 (Ultimate Edition)
Build #IU-242.23726.103, built on October 23, 2024

```
java.lang.Throwable: org.infernus.idea.checkstyle.util.Notifications <clinit> requests com.intellij.notification.NotificationGroupManager instance. Class initialization must not depend on services. Consider using instance of the service on-demand instead.
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:376)
	at com.intellij.serviceContainer.ComponentManagerImplKt.checkOutsideClassInitializer(ComponentManagerImpl.kt:1588)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1557)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:746)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:690)
	at com.intellij.notification.NotificationGroupManager.getInstance(NotificationGroupManager.java:19)
	at org.infernus.idea.checkstyle.util.Notifications.<clinit>(Notifications.java:20)
	at org.infernus.idea.checkstyle.startup.NotifyUserIfPluginUpdated.execute(NotifyUserIfPluginUpdated.java:36)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:496)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
```

**Comment by jshiell** [comment]
Thanks for the report! God I hate new Jetbrain's insistence on lazy-init for everything 😭 In any case, easy enough to fix this case, released in 5.97.0.

**Comment by ahus1** [comment]
Thank you for taking care of this one, I really appreciate it!
